docs/protocol/tokenomics.md

```diff
@@ -0,0 +1,101 @@
+
+Let us briefly present how the staking model for agents and component code incentivizes software developers:
+
+1. Developers stake agents and/or components on-chain by [minting they software packages as NFTs](./mint_packages_nfts.md#mint-a-component).
+2. Service owners use staked agents and components to create autonomous
+services. Then they [mint their service](./mint_packages_nfts.md#mint-a-service) in the Autonolas Protocol.
+3. Services can receive ETH donations through the Autonolas Protocol as a sign of recognition for their effort and utility.
+4. With a share of accrued donations, the protocol will reward the staked agents
+and/or components that facilitated such donations. Rewards are distributed proportionally to the code contribution to the ecosystem. Subject to governance approval, a portion of those donations can be eventually used by the DAO for ecosystem growth.
+5. Moreover, when the service owners own a certain threshold of veOLAS (e.g. they
+have locked a certain amount of OLAS for a certain period of time), they are
+considered whitelisted. The protocol will also distribute a share of the OLAS
+inflation to staked components (resp. agents) referenced in the whitelisted
+services.
+
+## Incentivizing bonders
+
+The Autonolas Tokenomics bonding mechanism allows users with enabled Uniswap V2 LP-pair assets to deposit them via the Autonolas depository smart contract and receive OLAS tokens at a discount relative to the price quoted on the relevant DEX after a vesting time. The interest rate on purchased bonds and the discount factor can be adjusted to incentivize or discourage bond demand. The protocol currently incentivizes bonding when there is a large potential output of code production in the ecosystem and a production function is used to measure potential code production during one epoch to establish the interest rate and discount factor.
+
+## Interacting with the protocol tokenomics
+
+The [Autonolas Tokenomics web app](https://tokenomics.autonolas.network/) provides a front-end for **donors**, **bonders** and **developers** to interact with the tokenomics model of the protocol.
+
+### Donate page
+
+Anyone can make a voluntary donation to a given service. Donations to the protocol are distributed to agent or component owners according to the direct contribution measure of them. Developers can later claim their incentive proportionally to the usefulness of their code.
+
+<figure markdown>
+![Tokenomics donate](./images/tokenomics_donate.png){ align=left width=500 }
+</figure>
+
+Connect your wallet and indicate the service ID you wish to donate. Allowed actions:
+
+* **Donate.** Donate the specified amount of ETH to the specified service. You can specify more than one service in a single call. The connected wallet will ask you to approve the transaction.
+
+    !!! warning "Important"
+        To avoid fragmented donations, the protocol currently only accepts donations of a minimum of 0.065 ETH.
```

Comment:
```suggestion
        To avoid fragmented donations, the protocol currently only accepts donations of at least 0.065 ETH. (If the donation is for several services, the total sum must be at least 0.065 ETH.)
```